src/http/representation/RepresentationMetadata.ts

```diff
@@ -257,6 +257,18 @@ export class RepresentationMetadata {
     return this;
   }
 
+  /**
+   * Verifies if a specific triple can be found in the metadata.
+   * Undefined parameters are interpreted as wildcards.
+   */
+  public has(
+    predicate: NamedNode | string | null = null,
+    object: NamedNode | BlankNode | Literal | string | null = null,
+    graph: MetadataGraph | null = null,
+  ): boolean {
+    return this.store.countQuads(this.id, predicate, object, graph) > 0;
```

Comment:
Yeah let's make an N3.js issue for that.

Reply:
`N3.js` has a `has` function actually because it implements the `DatasetCore` interface. But that one requires an exact triple without wildcards.

Reply:
No? https://github.com/rdfjs/N3.js/pull/275/files#diff-9f73dfbfc0fe486b1a0729bd957138af360c40f79ccaf2022673020565ce9c1cL279

Reply:
It's based on the interface defined here: https://rdf.js.org/dataset-spec/#datasetcore-interface . Quad is defined as being a NamedNode/BlankNode/Variable/Quad http://rdf.js.org/data-model-spec/#quad-interface. So only way to have wildcards there would be to use `Variable`s I guess?

Reply:
You could give secret `null`s, but yeah 😅
I'm currently working on that PR to have that functionality (fine for now).